sound/hybrid_font.h

```diff
@@ -300,6 +304,22 @@ class HybridFont : public SaberBase {
   void SB_Force() override { PlayCommon(&SFX_force); }
   void SB_Blast() override { Play(&SFX_blaster, &SFX_blst); }
   void SB_Boot() override { PlayPolyphonic(&SFX_boot); }
+
+  // Blaster effects, auto fire is handled by begin/end lockup
+  void SB_Stun() override { PlayPolyphonic(&SFX_stun); }
```

Comment:
Shouldn't this (and all the ones below) be PlayCommon instead of PlayPolyphonic?

Reply:
Ah yep. My bad. Will fix